nodes/ExtSlice.py

```diff
@@ -1,6 +1,21 @@
+"""
+ExtSlice astroid node
+
+Extended slicing which most python object like list and string don't support.
+
+Attributes:
+    - dims  (List[Node])
```

Comment:
Again, for the type to be consistent with the description, you'll probably want to write `List[Slice | Index]`